libs/components/forms/src/lib/modules/checkbox/checkbox-change.ts

```diff
@@ -4,7 +4,7 @@ import { SkyCheckboxComponent } from './checkbox.component';
 /**
  * Fires when users select or deselect the checkbox.
  */
-export class SkyCheckboxChange {
-  public source: SkyCheckboxComponent | undefined;
-  public checked: boolean | undefined;
+export interface SkyCheckboxChange {
+  source: SkyCheckboxComponent | undefined;
```

Comment:
Now that this is an interface I think it makes sense to use `?` instead of `| or undefined` for optional properties.

```
source?: SkyCheckboxComponent;
checked?: boolean;
```